SalesforceCommerceCloud/pwa-kit
# @W-14359655 - Fix Log out Auth loop

# Description



Fix a bug where the cookie removal was using different default attributes than the attributes used to set the cookie.

https://github.com/js-cookie/js-cookie#basic-usage

# Types of Changes




> Breaking changes include:
>
> - Removing a public function or component or prop
> - Adding a required argument to a function
> - Changing the data type of a function parameter or return value
> - Adding a new peer dependency to `package.json`

# Changes

- Updated cookie storage delete method.
- Added more error logs to ease debugging.

# How to Test-Drive This PR

1. Load https://runtime.commercecloud.com/mobify/blitz/adam/preview. Observe the cookie `cc-nx-g_RefArchGlobal` is set in the browser.
2. Login as a registered shopper in the PWA Kit storefront. Observe the cookie `cc-nx-g_RefArchGlobal` is removed from the browser and `cc-nx_RefArchGlobal` is created.
3. Log out. Observe there's no longer an auth loop. The cookie `cc-nx_RefArchGlobal` is removed and the cookie `cc-nx-g_RefArchGlobal` is set.
4. Observe no auth loop is happening in the Network tab.


# Checklists




## General


## Accessibility Compliance

You must check off all items in **one** of the follow two lists:


_or..._


## Localization
